src/commands/yubihsm/keys/import.rs

```diff
@@ -190,7 +190,7 @@ impl ImportCommand {
             key_id,
             label,
             DEFAULT_DOMAINS,
-            DEFAULT_CAPABILITIES,
+            DEFAULT_CAPABILITIES | yubihsm::Capability::EXPORTABLE_UNDER_WRAP,
```

Comment:
Perhaps it'd make sense to just add `EXPORTABLE_UNDER_WRAP` to the `DEFAULT_CAPABILITIES`.

If we did that, this would need to be updated too:

https://github.com/tendermint/kms/blob/69157e4adddead46b31fed8b032276c25975b32b/src/commands/yubihsm/keys/generate.rs#L76

...but right now the logic there is oddly inverted anyway (silly me), so it would actually be more straightforward I think to remove the `EXPORTABLE_UNDER_WRAP` capability if the key is explicitly configured to be non-exportable.